style.css

```diff
@@ -0,0 +1,71 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  max-height: 100vh;
+  font-family: 'Inter', sans-serif;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  color: #172b4d;
+}
+
+.header {
+  font-weight: 600;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+  padding: 0 36px 0 24px;
+  height: 50px;
+}
```

Comment:
Cheers for using flexbox 👍🏾 

- [ ] In the header let's have the hamburger menu on the right side. Currently, yours is on the left.
![image](https://user-images.githubusercontent.com/29071984/170606156-be6f1141-5806-46e4-b299-0474fee86a1a.png)